### DISCUSSION TITLE
Status field with conditions and a way to search for these statuses...

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#15552] [Archive]
In the status field, I have such a case in the Interface tab I have 2 statuses entered, while in the Display tab I have all listed. The remaining statuses are derived from the Conditions tab. It works great. The only problem is when I want to sort data by status, in the search engine I can choose through equals, I can only choose the 2 that I have entered in Interface, the rest does not appear. I cannot enter them in Interface because they only appear when the conditions are met, if I enter them, they are always visible.
Am I doing something wrong? I would like to see all statuses in my search
Directus 9.12.1

**Comment by br41nslug** [comment]
Only the options added to the main interface will be fetched for the search unfortunately. So for that to work you'd have to add all possible options to the main interface which can be problematic as i don't think there is a way to delete options from a condition

**Reply by koksikus** [reply]
Thanks for the answer. If I add to the interface then the statuses are duplicated, I have them twice. There must be conditions because, for example, you can choose published status only if other specified fields are filled. If the search was done with data not from the interface tab, but from the display tab, it would not be a problem, because in display I used all statuses to mark them properly.
If it was possible in the conditions to be set, hide this and that status and show this and that ...
In the latest version of Directus, it cannot be set this way either.

**Reply by koksikus** [reply]
Can we speak of a bug here or rather it should be an improvement. Much better if you can choose your own status when you search than you have to type it each time...
